public-src/phphants/scss/style.scss

```diff
@@ -499,18 +503,23 @@ ul#videos .video {
     }
 }
 
-ul#team li {
+ul#team li, ul#past-leaders li {
```

Comment:
This css addition implies new markup added, but I don't see it? Either add past leaders or drop this unused definition :)